projects/plots/plots/legacy/main.py

```diff
@@ -92,7 +92,13 @@ def main(
     foreground = RecoveredInjectionSet.read(foreground)
     # Filter unphysical waveforms generated by IMRPhenomXPHM glitch
     # https://git.ligo.org/reed.essick/rpo4-injection-triage/-/blob/main/rpo4a/README.md?ref_type=heads # noqa
-    foreground = foreground[foreground.snr < 1e4]
+    mask = foreground.snr > 1e4
+    num_unphysical = sum(mask)
+    if num_unphysical > 0:
```

Comment:
sorry for nit pick, but I dont think we even need the if statement here right?

Reply:
I felt like this was the kind of logging statement that could cause confusion if it reported 0 waveforms removed

Reply:
sounds good